osmosis/assetlist.json

```diff
@@ -543,6 +543,27 @@
         }
       ]
     },
+    {
+      "description": "Margined Power Token SQOSMO",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1g8qypve6l95xmhgc0fddaecerffymsl7kn9muw/squosmo",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1g8qypve6l95xmhgc0fddaecerffymsl7kn9muw/squosmo",
+      "name": "SQOSMO",
+      "display": "sqosmo",
+      "symbol": "SQOSMO",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/SQOSMO.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/SQOSMO.svg"
+        }
+      ]
+    },
```

Comment:
Please define a trace to OSMO

Reply:
I'm not sure its really a true trace to OSMO.  It's more of a wacky derivative on top

Reply:
I think we'll skip defining the trace for now but I want to add a new trace type: 'derivative' allowing for multiple origin assets, accommodiating for classic leverage (e.g., 3x); this 'power leveraged'; and also index funds, aggregating from multiple tokens.